utils/python/CIME/SystemTests/README

```diff
@@ -1,11 +1,13 @@
 The following are the test functionality categories:
   1) smoke tests
-  2) restart tests
-  3) threading/pe-count modification tests
-  4) sequencing (layout) modification tests
-  5) multi-instance tests
-  6) performance tests
-  7) spinup tests (TODO)
+  2) basic reproducibility tests
```

Comment:
I have updated the README in addition to the header documentation in config_tests.xml. We should probably pick one or the other to delete, to avoid redundancy.